packages/hooks/src/useCreateNFT.ts

```diff
@@ -250,6 +250,15 @@ export default function useCreateNFT(
             createLazyMintedAsset?.asset,
             ErrorMessages.ASSET_LAZY_MINT_CREATION_FAILED,
           )
+          setActiveProcess(CreateNftStep.OWNERSHIP)
+          // poll the api until the ownership is updated
+          console.info('polling api to check ownership...')
+          await pollOwnership({
+            assetId: createLazyMintedAsset.asset.id,
+            ownerAddress: account.toLowerCase(),
+            initialQuantity: '0',
+          })
+          console.info('polling done')
```

Comment:
Instead of duplicating the pollOwnership logic, we could move the creation of the "normal" asset logic in a `else` and set the created asset id outside the if/else with a `let`.

Reply:
Yes I even wanted to split this function into 2 that return the asset id but wanted to keep things simple for now